### ISSUE TITLE
Suggested Pitch Bend workaround, for extended plugins

### ISSUE THREAD

**Issue by Corshi** [LMMS/lmms#5740]
### Suggested Pitch Bend workaround for extended plugins:

**## Enhancement Summary**
Extension to save notes/sounds in formats that have Pitch Bend.
With user friendly interface, for it.

**### Justification**
It is annoying to make "New Project", every time, I want to "Pitch Bend" a note/sound.
Expectantly in middle of the work.
Super super anoing for sf2/gig and vst guitars.  
>>But I still love this project 😃 <<

**#### Mockup**
1.	Just like “Controller Rack” create “Sound Pad”.
2.	In “Piano-Roll” window, add option under Right Mouse Click “Add to Sound Pad”, if clicked on crated note.
3.	Make a default directory, under “documents/Imms” named “sound_pad”
4.	Files added to “Sound Pad”, should be a single sound files saved as .wav, (or compressed if chosen in settings) in directory from point 3.
5.	Dragging files from “Sound Pad” should work exactly as dragging similar file types from “My Samples” section.
 
Also:
If possible, Default Naming Pattern for notes “[short_for_plugin]-[object_name]-[instrument_number]-[note]-[length]-[Next_Number]”
short_for_plugin = sf2, gig, vst, etc.
object_name = name of used “sf2, gig, vst, etc.”
instrument_number = number get from used “sf2, gig, etc.”
note = C1, …
Length = 1, …
Next_Number = just in case if similar name already exists

**Comment by Spekular** [comment]
I don't think this is the way to go. It doesn't save much effort compared to just soloing the note and exporting it into your samples directory. If we really want to support per-note pitch bends for monophonic instruments I think we should run several instances of it that are intelligently spawned from one piano roll pattern. If we want better resampling support I think that should be via a freeze function or something similar, and should make use of the zipped project format (that we'll eventually have) to ensure that the frozen track is part of the project file (and thus won't be deleted or moved by the user, breaking things).

**Comment by Corshi** [comment]
It does saves time if you are used to key-binding-sounds and are too "amateur" to afford said sound-pad 😊

But yeah, your point is valid.
Another +idea:
If we would be able to open more the one project at once AND copy “instruments” between them.
+connecting that to your "intelligently spawned from one piano roll" idea. 
Not only would that solve some problems, but also, make use of multi-screen setups.
Pure sounds laboratory 😊

Also,
Sometime after creating main post I came back to Audacity (also Free tool). Recording audio and edit it is easy (many plugins and save formats), but hardware is important for that process.
So it is a medium tier solution.
For now.
